contrib/packaging/docker/Dockerfile.runtime

```diff
@@ -88,6 +97,12 @@ cd /go/src/github.com/google/gops && \
 git checkout -b v0.3.6 v0.3.6 && \
 go install && \
 strip /go/bin/gops
+#
+# bpf-map
+#
+RUN go get -d github.com/cilium/bpf-map && \
+cd /go/src/github.com/cilium/bpf-map && \
+go build && strip bpf-map
```

Comment:
Why this addition? This is pointing to master and not to a specific version like it was set on lines 66-68 from the previous version of this file.

Reply:
It looks like this was moved from above. @nickb937 what was the reason to move this here. Also this adds another layer due to the additional `RUN`.

Reply:
IIRC, the bpf-map was moved here because it needed `go` to build from sources, which wasn't available in its previous location.